src/styles/blocks/days.scss

```
@@ -0,0 +1,28 @@
+.calendar__day {
+  @include square($sizeCardDay);
+  background-color: $colorCardDay;
+  border: 1px solid black;
+  position: relative;
+  cursor: pointer;
+  transition-property: transform, background-color;
+  transition-duration: 500ms;
```

use variable for this
```suggestion
  transition-duration: $transitionDuration;
```